package.json

```diff
@@ -16,7 +16,7 @@
     "build": "ember build",
     "start": "ember server",
     "test": "ember test",
-    "test:sauce": "ember sauce:connect && ember test --config-file testem.sauce.js --test-port 7000",
+    "test:sauce": "ember test --config-file testem.sauce.js --test-port 7000",
```

Comment:
Is sauce:connect not needed?

Reply:
I moved it to the `before_script` hook so that it aligns with the docs: https://github.com/johanneswuerbach/ember-cli-sauce#using-on-travis. I can put it back if there was a strong reason for having it there.